docs/src/API/Visualize.md

```diff
@@ -0,0 +1,29 @@
+# Visualize
+
```

Comment:
We provide some simple plotting features to help users diagnose convergence in the algorithm. Through
- Visualizing Priors 
- Visualizing slices of parameter ensembles over iterations (or algorithm time) over input space
- Visualizing the loss function or other computed error metrics from `compute_error!`

_The following documentation provides the overview of what is currently implemented for  `Plots` or `Makie` backends, these will be expanded in due course_